src/app/components/pages/history/history.component.scss

```diff
@@ -91,13 +91,13 @@
       color: rgba(30, 34, 39, 0.5);
       display: inline-block;
       font-size: 13px;
-      line-height: 15px;
-      margin: 0 4px;
+      margin: 4px 10px 0;
```

Comment:
What is this change for? With this the address and the QR code button are not aligned:
![al](https://user-images.githubusercontent.com/34079003/42170644-2448d78c-7de5-11e8-8f63-ce2271aa89dd.png)
Maybe it's better to modify `p` and `img` so that they are like in the desktop wallet.